Abstract Syntax and Semantics Document/index.html

```diff
@@ -398,9 +398,89 @@
+      </p>
+      
+      <aside class="example" title="Union of RDF Streams that Share Blank Nodes">
+        <pre><code class="highlight">
+        :g1 {_:1 :isIn :RedRoom. _:2 :isIn :RedRoom}    {:g1, prov:generatedAtTime, t1}
+        :g2 {_:1 :isIn :BlueRoom. }                     {:g2, prov:generatedAtTime, t2}
+        :g3 {_:3 :isIn :RedRoom. }                      {:g3, prov:generatedAtTime, t3}
+        </code></pre>
+        is a union of the following two streams:
+        <pre><code class="highlight">
+          :g1 {_:1 :isIn :RedRoom. _:2 :isIn :RedRoom}  {:g1, prov:generatedAtTime, t1}
+          :g3 {_:3 :isIn :RedRoom. }                    {:g3, prov:generatedAtTime, t3}
+        </code></pre>
+        <pre><code class="highlight">
+          :g2 {_:1 :isIn :BlueRoom. }                   {:g2, prov:generatedAtTime, t2}
+        </code></pre>
+      </aside>
+
+      <p>
+      <em>RDF stream merge</em> is a trinary relation on RDF streams. Let <code>S1</code>, <code>S2</code>, and <code>S3</code> be RDF streams.
+      We say <code>S3</code> is an <em>RDF stream merge</em> of <code>S1</code> and <code>S2</code> if and only if 
+      there exist streams <code>S4</code>, <code>S5</code>, and <code>S6</code>, with <code>S4</code> isomorphic to <code>S1</code>, <code>S5</code> isomorphic to <code>S2</code>, <code>S6</code> isomorphic to <code>S3</code>,
+      such that <code>S4</code> and <code>S5</code> have no blank nodes in common, and <code>S6</code> is the union of <code>S4</code> with <code>S5</code>.</p>
+      <p class="note">
+        Merge is the appropriate relation to consider when joining streams that are considered to not share blank nodes. Informally, they would be on different RDF surfaces.
+      </p>
+      
+      <aside class="example" title="Merge of RDF Streams that Do Not Share Blank Nodes">
+        <pre><code class="highlight">
+        _:1 {:axel :isIn :RedRoom. :darko :isIn :RedRoom} {_:1, prov:generatedAtTime, t1}
+        _:2 {:axel :isIn :BlueRoom. }                     {_:2, prov:generatedAtTime, t2}
+        _:3 {:minh :isIn :RedRoom. }                      {_:3, prov:generatedAtTime, t3}
+        </code></pre>
+        is a merge of the following two streams:
+        <pre><code class="highlight">
+        _:1 {:axel :isIn :RedRoom. :darko :isIn :RedRoom} {_:1, prov:generatedAtTime, t1}
+        _:2 {:minh :isIn :RedRoom. }                      {_:2, prov:generatedAtTime, t3}
+        </code></pre>
+        <pre><code>
+        _:1 {:axel :isIn :BlueRoom. }                     {_:1, prov:generatedAtTime, t2}
+        </code></pre>
+      </aside>
+      
+      <p class="note">Given any three RDF streams, the determination of their satisfaction of the union or merge relation is a purely syntactic computation. 
+        That is, it is not necessary to consider entailments of the RDF streams or to be concerned with the values of literals.</p>
+
+      <aside class="ednote">
```

Comment:
This discussion is in an ednote because it proposes an alternate, constructuve definition of the union and merge operation, and before this is include, even informatively, it should be proved to be equivalent to the normative definition, which is given above.